### PR TITLE
(titus) low hanging fruit. fix string comparison on the job type

### PR DESCRIPTION
**Pull request by jeyrschabu**

### CODE REVIEW

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java [hunk 1]
```diff
@@ -180,7 +180,7 @@ public String submitJob(SubmitJobRequest submitJobRequest) {
         if (jobDescription.getUser() == null) {
             jobDescription.setUser("spinnaker");
         }
-        if (jobDescription.getJobGroupSequence() == null && jobDescription.getType()!="batch") {
+        if (jobDescription.getJobGroupSequence() == null && jobDescription.getType().equals("batch")) {
```
Comment (tomaslin) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java]: this is not the same logic

Comment (tomaslin) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java]: needs to be !jobDescription.getType( ... or change batch to service

Comment (jeyrschabu) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java]: that snuck in there. fixing it

Comment (jeyrschabu) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java]: good catch!

**Comment by jeyrschabu** [comment]
@tomaslin PTAL

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java [hunk 2]
```diff
@@ -180,7 +180,7 @@ public String submitJob(SubmitJobRequest submitJobRequest) {
         if (jobDescription.getUser() == null) {
             jobDescription.setUser("spinnaker");
         }
-        if (jobDescription.getJobGroupSequence() == null && jobDescription.getType()!="batch") {
+        if (jobDescription.getJobGroupSequence() == null && !jobDescription.getType().equals("batch")) {
```
Comment (tomaslin) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java]: I would change the check to be .equals("service")

**Pull request merged**